modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -308,7 +318,11 @@ The OTLP gRPC exporter exports data using the OpenTelemetry protocol (OTLP).
 ----
 <1> The OTLP gRPC endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
 <2> The client side TLS configuration. Defines paths to TLS certificates.
-<3> Headers are sent for every RPC performed during an established connection.
+<3> Disables client transport security when set to `true`; `false` by default.
+<4> Skips verifying the certificate when set to `true`; `false` by default.
```

Comment:
```suggestion
<4> Skips verifying the certificate when set to `true`. The default value is `false`.
```

Reply:
The original text was suggested by @max-cx. There should be probably agreement how to write this correctly. 

For now I will accept the suggestion.